lib/coverband/tasks.rb

```diff
@@ -20,20 +20,23 @@
             require baseline_file
           end
         end
-        if defined? Rails
-          Dir.glob("#{Rails.root}/app/**/*.rb").sort.each { |file| 
+        if defined? Rails && !Rails.config.eager_load
```

Comment:
No - this means that if Rails is defined and `eager_load` is TRUE then Coverband wouldn't load anything itself (since it would all have been required by the rails boot process.)

`eager_load` is generally true in prod-like and test environments.